cmd/create/cluster/cmd.go

```diff
@@ -3555,6 +3556,9 @@ func buildCommand(spec ocm.Spec, operatorRolesPrefix string,
 			securitygroups.SgKindFlagMap["Control Plane"], strings.Join(spec.AdditionalControlPlaneSecurityGroupIds, ","))
 	}
 
+	if len(properties) > 0 {
+		command += fmt.Sprintf(" --properties %s", strings.Join(properties, ","))
```

Comment:
shouldn't this be multiple --properties invocations? I am not sure it works this way

Reply:
you are correct! let me fix that